renderer/src/components/GanttRow.tsx

```
@@ -400,14 +404,55 @@ const GanttRow = ({ index, style, data }: ListChildComponentProps<GanttRowData>)
           row.label
         )}
       </div>
-      <div className="gantt-timeline" style={{ width: data.timelineWidth }}>
+      <div
+        className="gantt-timeline"
+        style={{ width: data.timelineWidth }}
+        role="button"
+        tabIndex={0}
+        onClick={(event) => {
+          const rect = event.currentTarget.getBoundingClientRect();
+          const scrollLeft = data.getScrollLeft();
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)